repository.json

```diff
@@ -857,6 +870,63 @@
 				}
 			]
 		},
+		{
+			"name": "pydantic",
+			"description": "Data validation using Python type hints.",
+			"author": "samuelcolvin",
+			"issues": "https://github.com/pydantic/pydantic/issues",
+			"releases": [
+				{
+					"base": "https://pypi.org/project/pydantic",
+					"asset": "pydantic-*-py3-none-any.whl",
+					"python_versions": ["3.8"]
+				}
+			]
+		},
+		{
+			"name": "pydantic-core",
+			"description": "Core functionality for Pydantic validation and serialization.",
+			"author": "samuelcolvin",
+			"issues": "https://github.com/pydantic/pydantic-core/issues",
+			"releases": [
+				{
+					"platforms": ["windows-x32"],
+					"base": "https://pypi.org/project/pydantic-core",
+					"asset": "pydantic_core-*-cp38-none-win32.whl",
+					"python_versions": ["3.8"]
+				},
+				{
+					"platforms": ["windows-x64"],
+					"base": "https://pypi.org/project/pydantic-core",
+					"asset": "pydantic_core-*-cp38-none-win_amd64.whl",
+					"python_versions": ["3.8"]
+				},
+				{
+					"platforms": ["linux-x64"],
+					"base": "https://pypi.org/project/pydantic-core",
+					"asset": "pydantic_core-*-cp38-cp38-musllinux_1_1_x86_64.whl",
```

Comment:
We should probably rather rely on `manylinux` assets, which is also what ST core uses.

Reply:
it is musl linux?

Reply:
According to a recent comment ST uses glibc 2.13, which is "manylinux" in terms of wheel files, if that's what you are asking for. As such all dependencies should proably also use glibc platform.

Reply:
I thought `musllinux` is self-sustaining (portable) while glibc requires the host's glibc version. Will test later .

Reply:
Not sure about that, but I woudn't start mixing platforms. All libraries should use the same runtime, if possible.

Reply:
change to manylinux.